### PR TITLE
Fi 534 fix uscore test names

### PR DESCRIPTION
**Pull request by czh-orz**
This changes test names to be more accurate

**Submitter:**
- [X] This pull request describes why these changes were made 
- [X] Internal ticket for this PR: https://oncprojectracking.healthit.gov/support/browse/FI-534
- [X] Internal ticket links to this PR
- [ ] Internal ticket is properly labeled (Community/Program)
- [ ] Internal ticket has a justification for its Community/Program label
- [X] Code diff has been reviewed for extraneous/missing code
- [X] Tests are included and test edge cases
- [X] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: generator/uscore/uscore_generator.rb [hunk 1]
```diff
@@ -200,14 +200,16 @@ def create_include_test(sequence)
       end
 
       def create_revinclude_test(sequence)
+        first_search = find_first_search(sequence)
+        return if first_search.blank?
+
         revinclude_test = {
-          tests_that: "Server returns the appropriate resources from the following _revincludes: #{sequence[:revincludes].join(',')}",
+          tests_that: "Server returns valid Provenance resources from #{sequence[:resource]} search by #{first_search[:names].join(' + ')} + _revIncludes: Provenance:target",
```
Comment (Jammjammjamm) [inline generator/uscore/uscore_generator.rb]: This test doesn't validate the Provenance resources.

Comment (czh-orz) [reply generator/uscore/uscore_generator.rb]: removed 'valid' from description.
